Simplenote/src/main/java/com/automattic/simplenote/authentication/SimplenoteCredentialsActivity.java

```diff
@@ -83,8 +84,12 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             EditText inputPassword = ((TextInputLayout) findViewById(R.id.input_password)).getEditText();
 
             if (inputEmail != null && inputPassword != null) {
-                inputEmail.setText(BuildConfig.LOGIN_EMAIL);
-                inputPassword.setText(BuildConfig.LOGIN_PASSWORD);
+                if (!TextUtils.isEmpty(BuildConfig.LOGIN_EMAIL)) {
```

Comment:
I added this because during development it clears things out leading to confusion. Presumably we meant this value to only set if it had something.